start.sh

```diff
@@ -5,15 +5,28 @@ SERVICES_YAML_PATH=${SERVICES_YAML_PATH:-/nail/etc/services/services.yaml}
 CASSANDRA_CLUSTER_CONFIG=${CASSANDRA_CLUSTER_CONFIG:-/var/run/synapse/services/cassandra_spectre.main.json}
 # We run 1 worker per container in production
 WORKER_PROCESSES=${WORKER_PROCESSES:-1}
+NGINX_CONF=/code/config/nginx.conf
 
 if [ $ACCEPTANCE ]; then
     # Cassandra ip is automatically generated
     host=$(grep 'cassandra_spectre.main:' /nail/etc/services/services.yaml | cut -d' ' -f3 | cut -d',' -f1)
     echo '[{"name": "cassandra-spectre-itest","host": "'$host'","port": 9042,"id": 1,"weight": 10}]' > $CASSANDRA_CLUSTER_CONFIG
 fi
 
+if [ "$DISABLE_STDOUT_ACCESS_LOG" = "1" ]; then
```

Comment:
Rather than a boolean flag I'd just pass in the value you want access_log to be set to. That way it's more extensible and we won't have to change this again in the future

Reply:
Yes but it's trickier because we are not only talking about the destination but also the log format to use
eg
access_log off;
or 
access_log /dest/ format;

but totally doable if you think this will be useful in the future

Reply:
Oh we cannot have `access_log off format`?

Also I just realized that sed replaces the entire line, so it'd be hard to just replace a placeholder anyway.

I'm fine with the current flag then.